### PR TITLE
change description indicator field to markdown

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-598)

## Description
Change several indicator fields from `longText` to `markdown`

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.8.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No
   - [ ] need to check

### CODE REVIEW

**Comment by idovandijk** [comment]
Does server support changing indicator field type?

**Comment by GuyAfik** [comment]
> Does server support changing indicator field type?

from 6.8 yes its supported, however this PR is still not ready (hence its draft), need to do some tests.

**Comment by Ni-Knight** [comment]
@GuyAfik please change all "longtext" fields (its about 3) no need to leave them as they are.

This wont pass validation as it checks for changes in field type, @ShahafBenYakir - I believe we will have to force merge it :(

**Comment by GuyAfik** [comment]
> @GuyAfik please change all "longtext" fields (its about 3) no need to leave them as they are.
> 
> This wont pass validation as it checks for changes in field type, @ShahafBenYakir - I believe we will have to force merge it :(
> 
> Seems like the test playbook is failing on a race condition, the indicator is created but fetched too soon and returns empty, adding a small sleep step will solve it - #19932

done, let me know if anything else needs to be changed from `longText` to `markdown` among the indicatorFields

**Comment by GuyAfik** [comment]
seems like even following https://github.com/demisto/content/pull/19932, `Test IP Indicator Fields` still failing

File: Packs/CommonTypes/IndicatorFields/incidentfield-description.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
     "version": -1,
     "unmapped": false,
     "hidden": false,
-    "type": "longText",
+    "type": "markdown",
     "breachScript": "",
     "editForm": true,
-    "description": "",
+    "description": "re-install CommonTypes if the type is not markdown",
```
Comment (Ni-Knight) [inline Packs/CommonTypes/IndicatorFields/incidentfield-description.json:20]: Pretty sure this wont show up for the customer as it wont actually update this file, or did you check it?

Comment (GuyAfik) [reply Packs/CommonTypes/IndicatorFields/incidentfield-description.json:20]: seems like it doesn't work, you are right, removing it

**Review by Ni-Knight — APPROVED** [review]

**Review by JudahSchwartz — APPROVED** [review]

**Comment by JudahSchwartz** [comment]
@ShahafBenYakir this will need a force due to the incorrect validation

**Comment by GuyAfik** [comment]
@ShahafBenYakir build fails only on validation as @JudahSchwartz mentioned above.

image

```
[ERROR]: Packs/CommonTypes/IndicatorFields/incidentfield-groups.json: [IF111] - Changing incident field type is not allowed.
```

Feel free to force merge it, the PR is ready and good to go.

**Pull request merged**